_posts/2018-01-19-event-lineup.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "The 2018 Rust Event Lineup"
+author: Rust Community
+description: "Multitude of Rust event are happening this year; join us at one near you!"
+---
+
+Every year there are multiple Rust events around the world, bringing together the community.
```

Comment:
i think we could add a few more sentences to this intro. something like "last year we wrote [link to last year] this year we are hoping to write a few posts spotlighting events that we have across the world. here are a few that we know of so far! if you have an event please contact XXX"

depending on if this goes out before or after our goals RFCs for comm team and the larger roadmap post we might allude to how these events help the language achieve goals XYZ